storage/innobase/xtrabackup/src/xtrabackup.cc

```diff
@@ -2641,6 +2659,7 @@ static bool innodb_init(bool init_dd, bool for_apply_log) {
   err = srv_start(false, to_lsn);
 
   if (err != DB_SUCCESS) {
+    xb::error() << "Failed to srv_start ";
```

Comment:
I think it doesn't add value. Can be removed.

Reply:
fixed